src/Nethermind/Nethermind.Evm/TransactionProcessor.cs

```diff
@@ -284,6 +284,11 @@ private void Execute(Transaction transaction, BlockHeader block, ITxTracer txTra
                         {
                             throw new OutOfGasException();
                         }
+                        
+                        if (spec.IsEip3541Enabled && substate.Output.Length >= 1 && substate.Output.Span.Slice(0, 1)[0] == 0xEF)
```

Comment:
StartsWith?

Reply:
this is byte[]
not sure about why Span is needed

Reply:
@tkstanczak This is not byte[]. This is ReadOnlyMemory<byte>. If you know a more elegant way to check it, please let me know.
@LukaszRozmej Thx, I changed it to the extensions method.